### PR TITLE
Imp wave2 844 grad details important facts

### PR DESCRIPTION
**Pull request by mhughes2012**
Important facts implementation

### CODE REVIEW

File: sites/all/modules/ucalgary/ucgy_fgs/templates/node--fgs-graduate-degree.tpl.php [hunk 1]
```diff
@@ -60,6 +60,61 @@
...
+  if (isset($content['field_fgsif_courses']) || isset($content['field_fgsif_supervisor'])
+  || isset($content['field_fgsif_commitment']) || isset($content['field_fgsif_class_delivery'])
+  || isset($content['field_fgsif_completion']) || isset($content['field_fgsif_awards_and_funding']) ) { ?>
+  <div class="fgs-important-facts">
+     <h2>Important Facts</h2>
+
+    <ul class="facts">
+      <?php if (isset($content['field_fgsif_courses'])) { ?>
+        <li class="courses">
+          <p><strong><?php echo t('Courses')?></strong></p>
```
Comment (dpfitzsi) [inline sites/all/modules/ucalgary/ucgy_fgs/templates/node--fgs-graduate-degree.tpl.php]: Should close PHP statement with semicolon.

File: sites/all/modules/ucalgary/ucgy_fgs/templates/node--fgs-graduate-degree.tpl.php [hunk 2]
```diff
@@ -60,6 +60,61 @@
...
+          <?php print render($content['field_fgsif_courses']);?>
+          <?php   if (isset($content['field_fgsif_listofcourses'])) { ?>
```
Comment (dpfitzsi) [inline sites/all/modules/ucalgary/ucgy_fgs/templates/node--fgs-graduate-degree.tpl.php]: Strange indenting/spacing.

File: sites/all/modules/ucalgary/ucgy_fgs/templates/node--fgs-graduate-degree.tpl.php [hunk 3]
```diff
@@ -60,6 +60,61 @@
...
+            <p><?php print render($content['field_fgsif_listofcourses']);?></p>
+          <?php } ?>
+        </li>
+      <?php } ?>
+      <?php if (isset($content['field_fgsif_supervisor'])) { ?>
+        <li class="supervisor">
+          <p><strong><?php echo t('Supervisor')?></strong></p>
```
Comment (dpfitzsi) [inline sites/all/modules/ucalgary/ucgy_fgs/templates/node--fgs-graduate-degree.tpl.php]: Should close PHP statement with semicolon.

File: sites/all/modules/ucalgary/ucgy_fgs/templates/node--fgs-graduate-degree.tpl.php [hunk 4]
```diff
@@ -60,6 +60,61 @@
...
+          <?php print render($content['field_fgsif_supervisor']);?>
+        </li>
+      <?php } ?>
+      <?php if (isset($content['field_fgsif_commitment'])) { ?>
+        <li class="commitment">
+          <p><strong><?php echo t('Commitment')?></strong></p>
```
Comment (dpfitzsi) [inline sites/all/modules/ucalgary/ucgy_fgs/templates/node--fgs-graduate-degree.tpl.php]: Should close PHP statement with semicolon.

File: sites/all/modules/ucalgary/ucgy_fgs/templates/node--fgs-graduate-degree.tpl.php [hunk 5]
```diff
@@ -60,6 +60,61 @@
...
+          <?php print render($content['field_fgsif_commitment']);?>
+        </li>
+      <?php } ?>
+      <?php if (isset($content['field_fgsif_class_delivery'])) { ?>
+        <li class="class-delivery">
+          <p><strong><?php echo t('Class delivery')?></strong></p>
```
Comment (dpfitzsi) [inline sites/all/modules/ucalgary/ucgy_fgs/templates/node--fgs-graduate-degree.tpl.php]: Should close PHP statement with semicolon.

File: sites/all/modules/ucalgary/ucgy_fgs/templates/node--fgs-graduate-degree.tpl.php [hunk 6]
```diff
@@ -60,6 +60,61 @@
...
+          <?php print render($content['field_fgsif_class_delivery']);?>
+        </li>
+      <?php } ?>
+      <?php if (isset($content['field_fgsif_completion'])) { ?>
+        <li class="completion">
+          <p><strong><?php echo t('Completion')?></strong></p>
```
Comment (dpfitzsi) [inline sites/all/modules/ucalgary/ucgy_fgs/templates/node--fgs-graduate-degree.tpl.php]: Should close PHP statement with semicolon.

File: sites/all/modules/ucalgary/ucgy_fgs/templates/node--fgs-graduate-degree.tpl.php [hunk 7]
```diff
@@ -60,6 +60,61 @@
...
+          <?php print render($content['field_fgsif_completion']);?>
+        </li>
+      <?php } ?>
+      <?php if (isset($content['field_fgsif_awards_and_funding'])) { ?>
+        <li class="awards-funding">
+          <p><strong><?php echo t('Awards and funding')?></strong></p>
```
Comment (dpfitzsi) [inline sites/all/modules/ucalgary/ucgy_fgs/templates/node--fgs-graduate-degree.tpl.php]: Should close PHP statement with semicolon.

**Comment by mhughes2012** [comment]
thank @dpfitzsi, I updated the indenting and closed off the t statements where needed.

**Comment by mhughes2012** [comment]
@gsowrey merge conflicts for this PR resolved.

**Comment by mhughes2012** [comment]
@gsowrey conflicts resolved.

**Comment by mhughes2012** [comment]
This PR is having issues. Closing this one and making a new one. New PR #490

**Pull request closed without merging**